src/backend/sev/snp/firmware/mod.rs

```diff
@@ -57,6 +57,14 @@ bitflags! {
 pub struct Identifier(Vec<u8>);
 
 impl Identifier {
+    // Get a unique identifier for the VCEK key to be used in linux file names
+    pub fn vcek_cache_path(&self, version: &TcbVersion) -> String {
+        format!(
+            "/var/cache/amd-sev/Milan-{:x}-{:02}-{:02}-{:02}-{:02}",
```

Comment:
We can't hard code `Milan` here.

Reply:
@npmccallum  It's already hardcoded in the URL

Reply:
@haraldh What happens when Genoa is released?

Reply:
@haraldh My understanding is that we have invented this filename format. I'm not sure why the generation name is even needed here. Is it to avoid collisions after a CPU swap?

Reply:
Prefixed with `vcek-` now.